x-pack/plugins/reporting/common/types.ts

```diff
@@ -57,22 +57,29 @@ export interface ReportOutput {
 
 export type TaskRunResult = Omit<ReportOutput, 'content'>;
 
+export interface BaseParams {
```

Comment:
Moved this interface from `server` so it can replace an inline definition of the same interface